### ISSUE TITLE
Adding collaborator via link

### ISSUE THREAD

**Issue by pavanjoshi914** [CircuitVerse/CircuitVerse#2307]
**Is your feature request related to a problem? Please describe.**
Currently we can add members to the group via invitation link , which is token based invitation system and expires every 7 days .
Similar functionality can be implemented for add collaborators modal.

**Describe the solution you'd like**
Adding functionality to add collaborators via invitation links

**Are you working on this? (Yes/No)**
NO

**Comment by ShreyaPrasad1209** [comment]
@Shivansh2407 do you think we can add this to good first issues?

**Comment by Shivansh2407** [comment]
> @Shivansh2407 do you think we can add this to good first issues?

Yes! As we need a functionality which is similar to the one already used by CircuitVerse in group invitation, so we can push it to the good first issues. I think a fix including the invitation functionality with minor tweaks with respect to collaborators would be good to go.

**Comment by pavanjoshi914** [comment]
Contributor can refer this PR  and can understand the designing behind the feature https://github.com/CircuitVerse/CircuitVerse/pull/1997

**Comment by 19bcs6140** [comment]
Hi Everyone @pavanjoshi914 @ShreyaPrasad1209 @Shivansh2407  I'm Rahul from Chandigarh University, Chandigarh currently pursuing my bachelor's in Computer Science with AIML 3rd year . Recently I did an internship at ISRO and I am a web developer and Open source enthusiastics as I am Contribute to many small program and many project so now I think the time is coming to contribute to some big project so that's why I choose Circuitverse because Circuitverse is perfectly fit for me.  I would like to work on this invitation link if there is no assigned to this feature as of now.

**Comment by pavanjoshi914** [comment]
@sharmarahul1001  Go ahead!

**Comment by tabhay563** [comment]
@pavanjoshi914 I am new to coding and currently learning WEB D can you assign me this issue as this is open  after 19 days also can please elaborate more about this issue

**Comment by sharmarahul1001** [comment]
Hi @tabhay563 I'm @sharmarahul1001 currently I'm working  on this I m running late on schedule due to college exams, this issue is the same as #1997 but yet I will manage to complete it on time no worries

**Comment by harshalkaigaonkar** [comment]
open?

**Comment by vedant-jain03** [comment]
@tachyons  @pavanjoshi914 Can I work on this?

**Comment by pavanjoshi914** [comment]
> @tachyons @pavanjoshi914 Can I work on this?

Go ahead

**Comment by vedant-jain03** [comment]
> > @tachyons @pavanjoshi914 Can I work on this?
> 
> Go ahead

Sure

**Comment by Asterisk-Hunter** [comment]
is this issue resolved or can i work on it @vedant-jain03
